sdk/core/switcher/entry.S

```diff
@@ -311,6 +315,16 @@ exception_entry_asm:
 	reloadRegisters c1, cgp, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14, c15, csp
 	mret
 
+.Linvalid_entry:
+// Mark this threads as in the middle of a forced unwind.
```

Comment:
Can you expand this to explain *why* (we want to resume rather than unwind if the compartment doesn't have an error handler)?